src/inc/eventtracebase.h

```diff
@@ -216,11 +300,21 @@ extern "C" {
 #endif //!DONOT_DEFINE_ETW_CALLBACK && !DACCESS_COMPILE
 
 #include <clretwallmain.h>
+
+#elif defined(PLATFORM_UNIX)
+
+#include "clrallevents.h"
```

Comment:
clrallevents.h should probably be called something a bit more descriptive like clrlttngtracepoints.h so that it's clear that this file is LTTng specific.